book/src/configuration.md

```diff
@@ -16,7 +16,7 @@ To override global configuration parameters, create a `config.toml` file located
 | `middle-click-paste` | Middle click paste support. | `true` |
 | `scroll-lines` | Number of lines to scroll per scroll wheel step. | `3` |
 | `shell` | Shell to use when running external commands. | Unix: `["sh", "-c"]`<br/>Windows: `["cmd", "/C"]` |
-| `line-number` | Line number display (`absolute`, `relative`) | `absolute` |
+| `line-number` | Line number display: `absolute` simply shows each line's number, `relative` shows the distance from the current line, and `dynamic` switches between the two based on whether the editor is unfocused or in insert mode. | `absolute` |
```

Comment:
Should we call it hybrid like the original article rather than dynamic?

Reply:
Hybrid refers to the behavior added (unconditionally, not that it bothers me) in #656, not this contribution.